pkg/auth/authinfo/impl.go

```diff
@@ -40,6 +42,13 @@ func mapToList(list map[string]struct{}) []string {
 	return sortedList
 }
 
+func writeList(writer io.Writer, list []string, prefix, space, postfix string) {
```

Comment:
I rather have a convertsion to string... than a writelist.... because it will allow to use templates. I know you like prints... but templates provide defense in depth.

Reply:
Like this?